src/System.Net.Ping/tests/FunctionalTests/PingTest.cs

```diff
@@ -836,5 +836,14 @@ public async Task SendPingToExternalHostWithLowTtlTest()
             pingReply = await ping.SendPingAsync(host, TestSettings.PingTimeout, TestSettings.PayloadAsBytesShort, options);
             Assert.NotEqual(IPStatus.Success, pingReply.Status);
         }
+
+        [Fact]
+        [OuterLoop]
+        public void Ping_TimedOutSync_Success()
+        {
+            var sender = new Ping();
+            PingReply reply = sender.Send("192.0.2.0");
```

Comment:
is there any constant you should use instead of this address?

Reply:
Moved this into a setting constant rather than a literal to make it more self-documenting :)